examples/abitest/tests/src/tests.rs

```diff
@@ -52,7 +52,7 @@ fn test_abi() {
         FRONTEND_ENTRYPOINT_NAME,
     );
 
-    let (runtime, entry_channel) = oak_runtime::Runtime::configure_and_run(configuration)
+    let (runtime, entry_channel) = oak_runtime::configure_and_run(configuration)
```

Comment:
Does `examples/abitest/tests/Cargo.toml` need a `std` feature specifier, or does that come automatically (because `no_std` *isn't* specified)?

Reply:
I had previously added `std` as a default feature to `oak_runtime/Cargo.toml`, without it I think you are right that `examples/abitest/tests/Cargo.toml` would need to specify it.

Reply:
Got it, thanks.